clients/benchmarks/CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ target_include_directories( client
     $<BUILD_INTERFACE:${Boost_INCLUDE_DIRS}>
 )
 
+link_libraries( ${BOOST_ROOT}/lib )
```

Comment:
I'm looking into this; i don't want to use link_libraries(), which would cause problems down the road.